### PR TITLE
Update obtaining-a-root-oid-from-an-iso-name-registration-authority.md

### PR DESCRIPTION
**Pull request by HerbertMauerer**
update to make it clear there is no registration of OIDs with Microsoft.

### CODE REVIEW

**Comment by PRMerger3** [comment]
@HerbertMauerer : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: desktop-src/AD/obtaining-a-root-oid-from-an-iso-name-registration-authority.md [hunk 1]
```diff
@@ -13,12 +13,4 @@ The preferred way to obtain a root object identifier (OID) is to request one fro
...
-Regardless of the source used to get the OID, if you intend to extend the Active Directory schema and wish to apply for the Certified for Windows logo, you must register your OID with Microsoft. For more information about how to register your OID with Microsoft, see [Obtaining an Object Identifier from Microsoft](obtaining-an-object-identifier-from-microsoft.md).
-
- 
-
- 
-
-
-
-
+Microsoft offers a script to generate a unique OID, which is not registered with a registration authority. This approach is sufficient for schema extensions you use in your own forests, where no external parties inquires about the OID registration. For more details on how to create such a base OID, see [Obtaining an Object Identifier from Microsoft](obtaining-an-object-identifier-from-microsoft.md).
```
Comment (alvinashcraft) [inline desktop-src/AD/obtaining-a-root-oid-from-an-iso-name-registration-authority.md]:
```suggestion
Microsoft offers a script to generate a unique OID, which is not registered with a registration authority. This approach is sufficient for schema extensions you use in your own forests, where no external parties inquire about the OID registration. For more details on how to create such a base OID, see [Obtaining an Object Identifier from Microsoft](obtaining-an-object-identifier-from-microsoft.md).
```

**Comment by PRMerger5** [comment]
@alvinashcraft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by alvinashcraft — APPROVED** [review]

**Comment by alvinashcraft** [comment]
Looks good. Thanks for your contribution! #sign-off

**Pull request merged**